tests/gnrc_ipv6_ext_frag/app.config

```diff
@@ -1,3 +1,3 @@
 # This test fails if the pool size is less than 3
-CONFIG_KCONFIG_MODULE_GNRC_IPV6_EXT_FRAG=y
+CONFIG_KCONFIG_USEMODULE_GNRC_IPV6_EXT_FRAG=y
```

Comment:
```suggestion
CONFIG_MODULE_GNRC_IPV6_EXT_FRAG=y
```
?

Reply:
Here we are activating the 'configurability' of the module via Kconfig.